src/utils/hooks/useLoadApplication.tsx

```diff
@@ -68,6 +71,15 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
 
       setApplication(applicationDetails)
 
+      const { id, code, name, startMessage } = application.template as Template
+
+      setTemplateType({
+        id,
+        code,
+        name: name as string,
+        startMessage: startMessage ? startMessage : undefined,
```

Comment:
```suggestion
        startMessage,
```
Pretty sure this does the same thing -- if it's undefined the property will carry through the "undefined" value, won't it?

Reply:
It can be `string` | `undefined` | `Null` => I'm just updating to be either `string` or `undefined`